src/components/BaseSearch.vue

```diff
@@ -54,6 +56,7 @@ function clearInput() {
       type="text"
       autocorrect="off"
       autocapitalize="none"
+      :autofocus="autofocus"
```

Comment:
This doesn't work if I open the modal again after closing it. It should work every time like with our `BaseInput` `focus-on-mount` prop